docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3.8
+WORKDIR /code
+RUN pip install git+https://github.com/dubizzle/python-github-backup.git#egg=github-backup
```

Comment:
Ideally you wanna do COPY . and pip install on the copied directory, if I doing docker on my branch and not master this will still build container for master